client/styles/components/_forms.scss

```diff
@@ -57,13 +57,18 @@
 
 .form__input {
   max-width: 90vw;
-  width: #{360 / $base-font-size}rem;
+  // width: #{360 / $base-font-size}rem;
+  width: 100%;
```

Comment:
This change has unintended consequences as it's used on all form inputs.  Check out the login and signup pages on desktop.  We might need an override that it specific to the account settings page.